template/config/storybook/config.js

```diff
@@ -0,0 +1,36 @@
+// import * as React from 'react';
+import { create } from '@storybook/theming';
+import { addParameters, configure } from '@storybook/vue';
+// import { withI18n } from 'storybook-addon-i18n';
```

Comment:
@taslangraham there are several lines commented out in this file, should they be there?

Reply:
I've removed them